### DISCUSSION TITLE
Just a question

### DISCUSSION THREAD

**Discussion by BabyBoySnow** [sigma67/ytmusicapi#404] [Q&A] [answered]
Hello, I was wondering if I could use this to see my most viewed songs on YouTube music? Sort of similar to Spotify wrap up?

**Comment by sigma67** [comment, marked as answer]
No, there is no views attribute on songs. You can only see what you can see on the web interface

This library is not some way to access hidden data. Only in very rare cases there is more data in the requests than what's shown in the interface
